src/SAML2/XML/saml/AuthnContextClassRef.php

```diff
@@ -69,6 +65,9 @@ public function setClassRef(string $classRef): void
      */
     public static function fromXML(DOMElement $xml): object
     {
+        Assert::same($xml->tagName, 'saml:AuthnContextClassRef');
```

Comment:
This should check `$xml->localName` instead. The namespace prefix can be anything, the standard does not mandate it to be `saml`.

Reply:
That goes for all of them I guess?

Reply:
Yep, we should always use `localName` when comparing.